run/websockets/test/e2e_test_cleanup.yaml

```diff
@@ -0,0 +1,22 @@
+steps:
+
+- id: 'Delete image and service'
+  name: 'gcr.io/cloud-builders/gcloud'
+  entrypoint: '/bin/bash'
+  args:
+    - '-c'
+    - |
+      ./test/retry.sh "gcloud container images describe gcr.io/${PROJECT_ID}/${_SERVICE}:${_VERSION}" \
+        "gcloud container images delete gcr.io/${PROJECT_ID}/${_SERVICE}:${_VERSION} --quiet"
+
+      ./test/retry.sh "gcloud run services describe ${_SERVICE} --region ${_REGION} --platform ${_PLATFORM}" \
+        "gcloud run services delete ${_SERVICE} --region ${_REGION} --platform ${_PLATFORM} --quiet"
+
+      ./test/retry.sh "gcloud compute networks vpc-access connectors describe my-connector --region ${_REGION}" \
+        "gcloud compute networks vpc-access connectors delete my-connector --region ${_REGION} --quiet"
+
+substitutions:
+  _SERVICE: websocket
```

Comment:
nit: websocket pluralization is inconsistent between directory name and service name.